src/simplewallet/simplewallet.cpp

```diff
@@ -1933,6 +1933,19 @@ bool simple_wallet::set_key_reuse_mitigation2(const std::vector<std::string> &ar
   return true;
 }
 
+bool simple_wallet::set_segregation_height(const std::vector<std::string> &args/* = std::vector<std::string>()*/)
+{
+  const auto pwd_container = get_and_verify_password();
+  if (pwd_container)
+  {
+    parse_bool_and_use(args[1], [&](bool r) {
```

Comment:
The expected value isn't bool but uint64_t, so `epee::string_tools::get_xtype_from_string()` should be used instead.